lib/deploy.js

```diff
@@ -351,7 +351,7 @@ function deployMQTrigger(mqType, name, namespace, topic, options) {
     apiVersion: 'kubeless.io/v1beta1',
     kind: `${mqType}Trigger`,
     metadata: {
-      name: helpers.escapeName(`${name}-${topic}`),
+      name: `${name}-${_.kebabCase(topic)}`,
```

Comment:
@andresmgot 
This will make 2 of the tests fail:
1. should deploy a function triggered by a topic on kafka
2. should deploy a function triggered by a topic on nats

both expect the function name "myFunction" to stay camelCase.

Reply:
that's fine, we can modify the tests to be adapted to the new code